handbook/engineering/security/secure-code-review.md

```diff
@@ -0,0 +1,41 @@
+# Requesting a security review
+
+Security code reviews differ from [peer code reviews](https://about.sourcegraph.com/handbook/engineering/code_reviews), as they are usually on large blocks of code, and after the fact. These reviews bring a second set of eyes, with dedicated focus to the problem.
+
+## Conducting a review
+
+To request a review from the security team:
+
+1. Create a GitHub issue in the [security repository](https://www.github.com/sourcegraph/security-issues), and add the *securityreview* label. 
+    * Provide a clear description, and links to the code, using Sourcegraph. 
+    * Provide as much detail as possible for the specifics of what has changed, and where we should focus. 
+    * Share usage scenarios, data flows, and any reference material
+
+1. Share the issue in the Security team slack channel, asking for a review. If you have a deadline, share it. By default, we target reviews for the upcoming iteration.
```

Comment:
Under what circumstances would a team have a deadline? How much lead time do you want?

Give teams some guidelines so they know how to think ahead.